static/redfish/v1/JsonSchemas/PowerSubsystem/PowerSubsystem.json

```diff
@@ -211,4 +211,4 @@
     "owningEntity": "DMTF",
     "release": "2021.2",
     "title": "#PowerSubsystem.v1_1_0.PowerSubsystem"
-}
```

Comment:
Why delete it and add it again?

Reply:
I don't know why it is showing really.. but , everything seems be OK to me..